### PR TITLE
fix annotsv phenotypes param+ fix annotSv output file bug

### PR DESCRIPTION
**Pull request by bartcharbon**
### Before merge:
- [x] Functionality works & meets specs
- [x] No issues running `bash test/pipeline_test.sh`
- [x] Code reviewed
- [n.a.] Documentation was updated

### After merge:
- [ ] Added feature/fix to draft release notes

## Notes by PR author
None.

### CODE REVIEW

File: pipeline_annotate.sh [hunk 1]
```diff
@@ -53,6 +53,12 @@ get_unique_phenotypes() {
   done
 }
 
+join_arr() {
```
Comment (dennishendriksen) [inline pipeline_annotate.sh:56]:
```suggestion
joinArr() {
```

Comment (dennishendriksen) [inline pipeline_annotate.sh:56]: Replace https://github.com/molgenis/vip/blob/master/pipeline_preprocess.sh#L86 and https://github.com/molgenis/vip/blob/master/pipeline_preprocess.sh#L160 with a call to joinArr

File: pipeline_annotate.sh [hunk 2]
```diff
@@ -403,14 +409,20 @@ executeAnnotSv() {
...
+
   if [ -n "${phenotypes}" ]; then
-    args+=("-hpo" "${phenotypes}")
-  fi
+    declare -A UNIQUE_PHENOTYPES
+    get_unique_phenotypes "${phenotypes}"
 
+    if [[ ${#UNIQUE_PHENOTYPES[@]} -gt 0 ]]
+    then
+      joined_phenotypes=$(join_arr , "${!UNIQUE_PHENOTYPES[@]}")
```
Comment (dennishendriksen) [inline pipeline_annotate.sh:422]: consistency, quote all strings:
```suggestion
      joined_phenotypes=$(join_arr "," "${!UNIQUE_PHENOTYPES[@]}")
```

Comment (dennishendriksen) [inline pipeline_annotate.sh:422]: make local and camel case.
```suggestion
      local joinedPhenotypes=$(join_arr , "${!UNIQUE_PHENOTYPES[@]}")
```

File: pipeline_annotate.sh [hunk 3]
```diff
@@ -53,6 +53,12 @@ get_unique_phenotypes() {
   done
 }
 
+join_arr() {
+  local IFS="$1"
+  shift
+  echo "$*"
```
Comment (dennishendriksen) [inline pipeline_annotate.sh:59]: other usages use `echo -e` instead of `echo`. which one is best?

File: pipeline_annotate.sh (hunk 1 above)
Comment (dennishendriksen) [inline pipeline_annotate.sh:56]: document function

**Review by dennishendriksen — CHANGES_REQUESTED** [review]

File: pipeline_annotate.sh [hunk 4]
```diff
@@ -403,14 +409,20 @@ executeAnnotSv() {
 
   args=()
   args+=("-SVinputFile" "${inputFilePath}")
-  args+=("-outputFile" "${outputFilePath}")
+  args+=("-outputFile" "${outputFilePath}.tsv")
```
Comment (dennishendriksen) [inline pipeline_annotate.sh:412]: add the .tsv in the call to the output so that the caller known the name of the output

**Review by dennishendriksen — CHANGES_REQUESTED** [review]

File: pipeline_annotate.sh (hunk 4 above)
Comment (bartcharbon) [reply pipeline_annotate.sh:412]: not sure what you mean

Comment (dennishendriksen) [reply pipeline_annotate.sh:412]:
```
currentOutputFilePath="${currentOutputDir}/${outputFilename}.tsv"
mkdir -p "${currentOutputDir}"
executeAnnotSv "${currentInputFilePath}" "${currentOutputFilePath}" "${assembly}" "${phenotypes}"
```

```
args+=("-outputFile" "${outputFilePath}")
```

**Review by dennishendriksen — APPROVED** [review]

**Pull request merged**